CanvasColorSpaceProposal.md

```diff
@@ -125,8 +120,12 @@ In implementations in which a ``UNPACK_COLORSPACE_CONVERSION_WEBGL`` of ``BROWSE
 
 Note that, while convenient, this behavior is likely less efficient than specifying color conversion in ``ImageBitmapOptions``, where the color conversion may be done asynchronously and simultaneously with image decode.
 
+An mechanism for using ``ImageBitmapOptions`` to specify color space conversion will be added in the context of WebGL color space support, to support asynchronous and decode-time color space conversion for texture data upload.
+
 ### WebGPU
 
+_NOTE: This section is not finalized and is being fleshed out and reviewed in Khronos_
```

Comment:
Not in Khronos - in the W3C. May want to link to https://github.com/gpuweb/gpuweb . I'm not sure there's a pull request open for this yet. cc @kainino0x @Kangz